### PR TITLE
Reimplement EvenOdd fill strategy

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

**Review by BarbourSmith — APPROVED** [review]
That was a big one! I can't say I fully understood all the C++ stuff, but it looks good to me!

File: algorithm/cgal/EagerTransform.h [hunk 1]
```diff
@@ -33,7 +33,7 @@ int EagerTransform(Geometry* geometry, int count) {
       }
       case GEOMETRY_POLYGONS_WITH_HOLES: {
         Plane transformed_plane =
-            unitPlane(geometry->plane(nth).transform(transform));
+            unitPlane<Kernel>(geometry->plane(nth).transform(transform));
```
[the remark this answers is not in the document]
Comment (BarbourSmith) [reply algorithm/cgal/EagerTransform.h:36]: Haha that's hard to write in markdown

[the remark this answers is not in the document]
Comment (pentacular) [reply algorithm/cgal/EagerTransform.h:36]: Ah, it make a version of unitPlane specialized to work on this particular Kernel type.

[the remark this answers is not in the document]
Comment (BarbourSmith) [reply algorithm/cgal/EagerTransform.h:36]: Ah gotcha :)

**Pull request merged**